model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -51,7 +51,8 @@ class FindingsMatcherTest : WordSpec() {
     private val licenseFindings = mutableSetOf<LicenseFinding>()
     private val copyrightFindings = mutableSetOf<CopyrightFinding>()
 
-    override fun isolationMode() = IsolationMode.InstancePerLeaf
+    // Use InstancePerRoot to work around https://github.com/kotest/kotest/issues/5021.
+    override fun isolationMode() = IsolationMode.InstancePerRoot
```

Comment:
I guess we have the same issue in `FindingCurationMatcherTest`.

Reply:
Actually, `FindingCurationMatcherTest` passes with `override fun isolationMode() = IsolationMode.InstancePerLeaf` removed completely, as the `lateinit` varaibles get initialized by each test already anyway.

Reply:
> So, shouldn't we have the possibility of race conditions, if we remove this entirely?

I don't think so, [because](https://kotest.io/docs/framework/concurrency.html) "by default, Kotest will execute each test case sequentially using a single thread".

Reply:
Ah, I see. This one is not for kotest. The property is `kotest.framework.parallelism`.